driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/hooks/DriverUiHookDesktop.java

```diff
@@ -43,6 +45,7 @@
     public void init() {
         WebDriverManager.registerWebDriverFactory(new DesktopWebDriverFactory(), browsers);
         GuiElement.registerGuiElementCoreFactory(new DesktopGuiElementCoreFactory(), browsers);
+        RetryAnalyzer.registerAdditionalRetryAnalyzer(new SeleniumRetryAnalyzer());
```

Comment:
Whats about the WebDriverRetryAnalyzer? It should also a default.

Reply:
https://github.com/telekom/testerra/blob/refactoring/retry-analyzer/driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/hooks/DriverUiHook.java#L76